src/common/comp/knowhere_config.cc

```diff
@@ -24,21 +24,27 @@
 #ifdef KNOWHERE_WITH_RAFT
 #include "index/ivf_raft/set_pool.h"
 #endif
+
 namespace knowhere {
 
 void
 KnowhereConfig::ShowVersion() {
 #define XSTR(x) STR(x)
 #define STR(x) #x
+    std::string msg = "Knowhere Version: ";
 #ifdef KNOWHERE_VERSION
+    msg = msg + XSTR(KNOWHERE_VERSION);
 #ifdef KNOWHERE_WITH_RAFT
-    LOG_KNOWHERE_INFO_ << "Knowhere Version: " << XSTR(KNOWHERE_VERSION) << "-gpu";
-#else
-    LOG_KNOWHERE_INFO_ << "Knowhere Version: " << XSTR(KNOWHERE_VERSION);
+    msg = msg + "-gpu";
 #endif
 #else
-    LOG_KNOWHERE_INFO_ << "Knowhere Version: unknown";
+    msg = msg + " unknown";
+#endif
+
+#ifdef _DEBUG
```

Comment:
AFAIK, `_DEBUG` is not in the standard, but `NDEBUG` is. Can you double-check?

Reply:
ok, updated